apps/site/lib/site_web/templates/event/index.html.eex

```diff
@@ -5,19 +5,29 @@
   </header>
```

Comment:
The calendar work does not play behave when event-page-setup script is run on this page.  (The conditional for the script is based on the `m-events-hub` selector, so using a selector from `_event_list` instead should do the trick.)

Reply:
Just updated this, and I'm having trouble reproducing so I'm hoping that means it's fixed now!

Reply:
sure is!